feature-libs/asm/components/customer-selection/customer-selection.component.spec.ts

```diff
@@ -240,4 +245,140 @@ describe('CustomerSelectionComponent', () => {
     ).toBeFalsy();
     expect(asmService.customerSearchReset).toHaveBeenCalled();
   });
+
+  describe('Search result navigation', () => {
+    beforeEach(() => {
+      spyOn(asmService, 'getCustomerSearchResults').and.returnValue(
+        of(mockCustomerSearchPage)
+      );
+      component.ngOnInit();
```

Comment:
You shouldn't need to invoke `ngOnInit` directly. `fixture.detectChanges` should trigger this

Reply:
i will re-test it...   I think i tested with fixture.detectChanges before but i will re-test

Reply:
i need to keep component.ngOnInit() it fails all test